TextFormatting.cs

```diff
@@ -21,14 +24,20 @@ public class TextFormatting : UdonSharpBehaviour
     [Tooltip("C# DateTime.Now.ToString formatting.")]
     public bool formatDateTime = false;
     [Tooltip("Replace Unicode characters with similar ASCII if available.")]
+    [Tooltip("Replace Unicode characters with similar ASCII if available. Requires Unidecoder for work.")]

```

Comment:
"Requires Unidecoder for work." should be "Requires Unidecoder to work."

Reply:
Done